src/resources/i18n/translations/en.json

```diff
@@ -17,16 +17,16 @@
   "Account": "Account",
   "Active Downloads": "Active Downloads",
   "Actual amount will be higher due to network fees": "The actual amount may be higher due to network fees",
-  "Add current item next in queue when playing a new item": "Add current item next in queue when playing a new episode",
-  "Add Custom RSS Feed": "Add custom RSS feed",
-  "Add custom RSS feeds": "Add custom RSS feeds",
+  "Add current item next in queue when playing a new item": "Start playback immediately before continuing",
```

Comment:
```suggestion
  "Add current item next in queue when playing a new item": "Continue playback after recently added items",
```

Reply:
Resolving to consolidate on other comment thread.